src/Components/Header/Header.js

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 import './Header.css';
+import { Link } from 'react-router-dom';
 
-const Header = () => {
+import SearchBar from '../SearchBar/SearchBar.js';
 
-}
+const Header = () => (
+  <div>
+    <h1>BreathEZ</h1>
+    <Link to='/faq'>FAQ</Link>
+    <SearchBar />
```

Comment:
It's possible that I'll want to re-organize this as the functionality progresses but I think we shouldn't worry about it now. For at least the next two days this is great.